src/api/search.ts

```diff
@@ -3,11 +3,13 @@ import Office from '../types/Office'
 import SearchMeta from '../types/SearchMeta'
 import apiClient from './client'
 
-export type SearchData = {
+export type SearchData<T> = {
```

Comment:
I think this SearchData should be deleted in favor of the new `SearchData.ts` definition in `src/types`